views/static-pages/research.njk

```diff
@@ -1,29 +0,0 @@
-{% from "components/hero.njk" import hero with context %}
```

Comment:
This is the old research landing page—which is now archived—so this page can go 🗑